src/components/DisplayCard/DisplayCard.tsx

```diff
@@ -77,7 +76,7 @@ export const DisplayCardTitle = ({
   return (
     <>
       <HStack paddingBottom="0.25rem">
-        <Text {...(styles.title as TextProps)} {...props}>
+        <Text {...(styles.title as TextProps)} mr="0" {...props}>
```

Comment:
i might be lacking context here but why did we make this change? the `props` allow for passing in `mr` already and this is a change that will affect **all** existing display card header components so it might be problematic

Reply:
>i might be lacking context here but why did we make this change?

Design wanted the dist from the header to the icon to be only 8px. Changing the text to have no right margin here. 

> the props allow for passing in mr already and this is a change that will affect all existing display card header components so it might be problematic

This is intentionally made as a broad change to allow all other display cards in the dashboard (Ie all display cards since they are not being used anywhere else, correct me if I am wrong about this) to follow this format instead